docker-compose.yml

```diff
@@ -24,8 +24,12 @@ boulder:
       - 9380:9380 # mail-test-srv
       - 9381:9381 # mail-test-srv
     links:
+      - bhsm:boulder-hsm
       - bmysql:boulder-mysql
       - brabbitmq:boulder-rabbitmq
+bhsm:
+    build: test/
```

Comment:
This image can be built from `boulder-tools` as a base but I think it does need to be rebuilt each time since it relies on a in tree file (`test/test-ca.key`) to run which may change. Moving the base to `boulder-tools` should significantly reduce the spin-up time.